src/replica/mutation_log.h

```diff
@@ -149,13 +149,10 @@ class mutation_log : public ref_counter
         return replay_block(log, callback, start_offset, end_offset);
     }
 
-    // Resets private-log with log files under `dir`.
-    // The original plog will be removed after this call.
-    // NOTE: private-log should be opened before this method called.
-    virtual error_code reset_from(const std::string &dir, io_failure_callback)
-    {
-        return ERR_NOT_IMPLEMENTED;
-    }
+    // Resets mutation log with log files under `dir`.
+    // The original log will be removed after this call.
+    // NOTE: log should be opened before this method called.
```

Comment:
I notice that old comments use `private log`, but this pr updates it into mutation log, will shared log use this function?

Reply:
I only think this should be a common function